app/permissions/decidim/department_admin/permissions.rb

```diff
@@ -161,6 +161,15 @@ def has_permission?(requested_action)
 
           # CONFERENCES
           -> { permission_for?(requested_action, :admin, :enter, :space_area, space_name: :conferences) },
+
+          # USERS ADMINISTRATORS
+          -> { permission_for?(requested_action, :admin, :enter, :space_area, space_name: :users) },
+          -> { permission_for?(requested_action, :admin, :read, :admin_user) },
+          -> { permission_for?(requested_action, :admin, :create, :admin_user) },
+          -> { permission_for?(requested_action, :admin, :read, :managed_user) },
+          # USERS PARTICIPANTS
+          -> { permission_for?(requested_action, :admin, :index, :officialization) },
+          -> { permission_for?(requested_action, :admin, :read, :officialization) },
```

Comment:
don't we need the permission to officialize a participant?

Reply:
@LauraJaimeC haves you checked that?

Reply:
yes! there is an other permission (create) under the read officialization. can you check it?